### ISSUE TITLE
Bug / Inconsistent error borders

### ISSUE THREAD

**Issue by mimacom-klier** [porscheui/porsche-ui-contribution#15]
**Description**  
The error border of input elements differs from the error border of the textarea element.

**To Reproduce**  
```
<section class="example">
  <label class="input input--error">
    <input
      class="input__field input__field--error"
      type="text"
      name=""
      value=""
      placeholder="Some input"
    />
    <span class="input__label">Some input</span>
  </label>
</section>

<section class="example">
  <label class="textarea">
    <textarea
      class="textarea__field textarea__field--error"
      placeholder=" Some textarea label"
    ></textarea>
    <span class="textarea__label"> Some textarea label </span>
  </label>
</section>
```

**Expected behavior**  
All input elements should have consistent error styling.

**Screenshots**  
![image](https://user-images.githubusercontent.com/43991732/58546760-5771c800-8206-11e9-9345-f97b4de17b35.png)

**UIKit version**
0.18.5

**Test Device (please complete the following information):**  
 - Device: [desktop]
 - OS: [macOS Mojave]
 - Browser [chrome]
 - Version [74]

**Comment by o-hook** [comment]
Please review PR: https://github.com/porscheui/porsche-ui-kit/pull/149

**Comment by o-hook** [comment]
Released with v0.18.6/core
